src/components/Flex/Flex.module.scss

```diff
@@ -0,0 +1,95 @@
+@import "../../styles/themes.scss";
+
+@mixin main-position-classes {
+  &Start {
+    justify-content: flex-start;
+  }
+
+  &End {
+    justify-content: flex-end;
+  }
+
+  &Center {
+    justify-content: center;
+  }
+
+  &SpaceBetween {
+    justify-content: space-between;
+  }
+
+  &SpaceAround {
+    justify-content: space-around;
+  }
+}
+@mixin secondary-position-classes {
+  &Start {
+    align-items: flex-start;
+  }
+
+  &End {
+    align-items: flex-end;
+  }
+
+  &Center {
+    align-items: center;
+  }
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  &.horizontal {
+    &Position {
+      @include main-position-classes;
+    }
+    &SpacingSize {
+      &Small {
+        & > * {
```

Comment:
instead of something so generic, we can use the `gap` property, it is more readable ?,i would accept a `number` prop and not preset, as this is a layout component and not all spacing is small, medium, large - we can have much much more